### PR TITLE
Update Kumascript in translation pt-BR

### PR DESCRIPTION
**Pull request by Fernandolrs**
Adjustment of every Kumascript project, translation into pt-BR.

### CODE REVIEW

File: macros/ArticlesByTag.ejs [hunk 1]
```diff
@@ -18,22 +18,26 @@ var l10n = {
         "en-US": "There have been no articles written yet. <a href=\"/en-US/docs/MDN/Getting_started\">Please consider contributing</a>.",
         "zh-CN": "这些文章尚未撰写。<a href=\"/zh-CN/docs/MDN/Getting_started\">欢迎贡献</a>。",
         "fr"   : "Aucun article n'a encore été écrit ou traduit. <a href=\"/fr/docs/MDN/Débuter_sur_MDN\">N'hésitez pas à contribuer.</a>",
-        "ru"   : "Вы ещё не написали статей. <a href=\"/ru/docs/MDN/Getting_started\">Пожалуйста, посмотрите о вкладе</a>."
+        "ru"   : "Вы ещё не написали статей. <a href=\"/ru/docs/MDN/Getting_started\">Пожалуйста, посмотрите о вкладе</a>.",
+        "pt-BR': "Não hà artigos escritos ainda. <a href=\"/pt-BR/docs/MDN/Primeiros_Passos\"> Por favor, considere contribuir </a>."
```
Comment (Elchi3) [inline macros/ArticlesByTag.ejs]: Here you've mixed single and double quotes `"pt-BR':`

Comment (Fernandolrs) [inline macros/ArticlesByTag.ejs]: Ok, I'll be reviewing all the submitted "Pull requests," and apologize for my inconvenience.

**Comment by jwhitlock** [comment]
1. I'm still planning on [bug 1340342](https://bugzilla.mozilla.org/show_bug.cgi?id=1340342) as the solution for translations.  It will take development work on each macro to update them, so this is an OK "for now" solution.
2. I still prefer one PR per macro, as stated on https://github.com/mozilla/kumascript/pull/142#issuecomment-309134227. It increases your chances of getting your changes merged.

**Comment by Fernandolrs** [comment]
Ok, thanks a lot, I'm still learning how to use GitHub, and the "pull requests".

The next submissions will be by macro, following your advice and project norms.

**Comment by Elchi3** [comment]
This is not merge-able as is and we're planning on a different way forward. I'm closing this to signal no further support to this approach from our side. Best is probably to follow https://bugzilla.mozilla.org/show_bug.cgi?id=1340342 and to stay tuned for a renewed KumaScript l10n infrastructure. Sorry for any inconveniences and thanks for supporting MDN.

**Comment by Fernandolrs** [comment]
OK no problems.
The MDN is growing, and needs constant updating, so as not to be obsolete.
If I can support MDN somehow I will.

**Pull request closed without merging**